README.md

```diff
@@ -184,6 +184,14 @@ Notes:
   * `"disk"`: Boot from local hard disk.
   * `"none"`: Reset boot order to default.
 
+### Checking a node's power status
+
+`GET /node/{node_id}/power_status`
+
+Notes:
+
+* Returns a JSON string that describes the node's power state.
```

Comment:
Should detail the format of the response, including possible values, and provide an example.